py/torch_tensorrt/fx/converters/operator.py

```diff
@@ -1207,7 +1208,7 @@ def add_slice(network, target, kwargs, name):
     stride = [1] * len(start)
     stride[dim] = step_int
     output_shape = list(input_val.shape)
-    output_shape[dim] = (stop_int - start_int) // step_int
+    output_shape[dim] = math.ceil((stop_int - start_int) // step_int)
```

Comment:
I think this needs to be `math.ceil((stop_int - start_int) / step_int)`, since the `ceil` is doing the int-casting